src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String PATTERN_FORMAT = "dd.MM.yyyy";
+
+    private static final int CONSTANT_NUM = 1;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(PATTERN_FORMAT);
+        LocalDate payPeriodStart = LocalDate.parse(dateFrom, formatter);
+        LocalDate payPeriodEnd = LocalDate.parse(dateTo, formatter);
+
```

Comment:
the same issue :) Let's remove all redundant empty lines

Reply:
I won't make empty lines anymore)